src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -37,6 +40,28 @@ public void initSendable(SendableBuilder builder) {
       builder.addDoubleProperty("CameraLatency", () -> {return latency; }, null);
   }
 
+  public void getVisionOutputs(){
+
+      VisionOutputs visionOutputs = new VisionOutputs();
+
+      var result = camera.getLatestResult();
+      Boolean cameraHasTargets = result.hasTargets();
+      visionOutputs.setCameraHasTargets(cameraHasTargets);
+
+      int tagIDs = target.getFiducialId();
+      visionOutputs.setTagIDs(tagIDs);
+
+      double latency = camera.getLatestResult().getLatencyMillis();
+      visionOutputs.setLatency(latency);
```

Comment:
It isnt good practice to create a variable, and then use it in only one place.  if latency will be used again later, this is the right way. But if this is the only place we will use it, the shorter  code

visionOutputs.setLatency ( camera.getLatestResult().getLatencyMillis() ) ; 

is preferred